### PR TITLE
[MDB IGNORE] [IDB IGNORE] feubrary tgu

### PR DESCRIPTION
**Pull request by Pepsilawn**
john wilard is asleep so this is a troll pr

cool contents:

- Abductors is playable
- Drones can build shit again
- janiborg rework
- thieves objective not being broken idk i only played it once and it was sheets busing me
- Holopay
- legion resprite
- maps minimum/maximum pop config actually working with votes (kilostation in rotation will be real)
- people no longer being deleted by disposal pipes
- metal foam qol
- new mob: brimdemons
- lizards gas
- my pr
- modular maintenance
- icebox cursed spring no longer spawning on station z level (kills ismael dimeling)
- some performance shit idk that shit looks complicated
- mob movement speed
- wizard being gibbed by rod form that was pretty funny though
- species uplink item @ebb-real
- something about a certain job's uniform resprite with one of the sexier one being overridden in Z_edits folder
- probably some other bullshit

bloodsucker code updated 2 match mutation undefining & incapacitated defining

All maps updated to reflect:
- Security locker room being an area
- wack ass changes brought by grand/lesser maintenance refactors
- paystand refactor
- sunflower refactor
- laundry room
- pubby updated 2 latest commit of the pr on map depot

### CODE REVIEW

**Comment by Pepsilawn** [comment]
why did that asshole confused react my pr

**Comment by HugoOdaX** [comment]
> why did that asshole confused react my pr

^ not john willard

**Comment by SpaceDragon00** [comment]
Helianthus personally failed ur linters with his maps I dont think hes coming to save u

**Comment by OneAsianTortoise** [comment]
Wow John switch to anime pfp

**Comment by Pepsilawn** [comment]
WTF are all these map errors none of them are from me

**Comment by Pepsilawn** [comment]
oh ok i get it they added check for apc var edits fuck you helianthus

**Comment by Helianthus00** [comment]
> @Helianthus00 there are people trolling on my pr do something

ok

**Comment by Pepsilawn** [comment]
YOU CLOSED IT IN THE MIDDLE OF THE LINTERS I WAS LOOKING AT THEM

**Comment by Pepsilawn** [comment]
i had to spend like 30 minutes fixing all of your APCs because they cause the checks to fail if they're offset by a value that isnt a multiple of 25 now

**Comment by JohnFulpWillard** [comment]
Have you tried an updatepaths

**Comment by Helianthus00** [comment]
> i had to spend like 30 minutes fixing all of your APCs because they cause the checks to fail if they're offset by a value that isnt a multiple of 25 now

Blame tg for making stupid arbitrary things that fail linters

**Comment by Pepsilawn** [comment]
> Have you tried an updatepaths

idk how i'd cherry pick APCs to change their var edited pos to ones that satisfy the checks i think using a text editor goes faster for that

**Comment by Pepsilawn** [comment]
I win, linters

**Comment by StoneMason** [comment]
"something about a certain job's uniform resprite idk what we're gonna do about that costume yet"

fulp_modules/Z_edits/erp_removal

**Comment by Pepsilawn** [comment]
ur so right

**Comment by Pepsilawn** [comment]
woops accidently shat all over my PR

**Comment by Pepsilawn** [comment]
FAKE merge conflict is preventing me from fixing my LINTERS

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -220,7 +220,7 @@
 		if(!C?.credits)
 			C?.RollCredits()
 //		C?.playtitlemusic(40)
-		C?.playcreditsmusic(40) // Fulp edit - Music
+		C?.playtitlemusic(40) // Fulp edit - Music
```
Comment (JohnFulpWillard) [inline code/__HELPERS/roundend.dm]:
```suggestion
		C?.playcreditsmusic(40) // Fulp edit - Music
```

**Review by JohnFulpWillard — COMMENTED** [review]
accidentally reversed a tg edit

**Pull request merged**